README.md

```diff
@@ -66,9 +77,10 @@ use {
 
     -- Snippets
     {'L3MON4D3/LuaSnip'},
-    {'rafamadriz/friendly-snippets'},
   }
 }
+-- Adding a snippets collection (Not required)
+use('rafamadriz/friendly-snippets'),
```

Comment:
The comment should say "-- Snippet collection (optional)". And put it back in the `requires` of lsp-zero, even if its optional its still related to lsp-zero.

Reply:
Done